velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -184,56 +180,29 @@ VectorFuzzer::Options getFuzzerOptions() {
 
 std::optional<CallableSignature> processSignature(
     const std::string& functionName,
+    const std::vector<TypePtr>& argTypes,
     const exec::FunctionSignature& signature) {
-  // Don't support functions with parameterized signatures.
-  if (!signature.variables().empty()) {
```

Comment:
On first glance this method looks like it can process any generic function signature, however its only used for non-templated/parameterized functions. I think it would be good to rename it to make it explicit and also keep the check here in case it ever gets used for a parameterized signatures. Maybe throw an exception or use a LOG(FATAL) so that it gets caught during development.